docs(k8s): add instructions for kubeconfig download from console (#5615)
scaleway/docs-content: pages/kubernetes/how-to/connect-cluster-kubectl.mdx

## Accessing the cluster

    <Tabs id="connect-cluster">
      <TabsTab label="Using the Scaleway console">
         1. Click **Kubernetes** in the **Containers** section of the [Scaleway console](https://console.scaleway.com) side menu. The Kubernetes Kapsule overview displays.
         2. From the drop-down menu, select the geographical region you want to manage.

>          3. Click the name of the cluster you want to access. The cluster overview page displays. This page provides several pieces of information about your cluster.

Reviewer: ```suggestion
         3. Click the name of the cluster you want to access. The cluster overview page displays. This page provides information about your cluster.
```

>          4. In the **Getting Started** section of the page click **2 Download kubeconfig**, then click **Generate kubeconfig**.

Reviewer: ```suggestion
         4. In the **Getting Started** section of the page, click **2 Download kubeconfig**, then **Generate kubeconfig**.
```

         5. Enter your existing Scaleway API key or click **Generate API key** to generate a new one.

>          6. (Optional) Enter API key specifics by select the API key bearer, enter a description and expiration date for the key. Then click **Generate API key**.

Reviewer: ```suggestion
         6. (Optional) Enter API key specifics by selecting the API key bearer, and entering a description and expiration date for the key. Then click **Generate API key**.
```
